EIPS/eip-3220.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 3220
+title: Crosschain Identifier Specification
+author: Weijia Zhang (@weijia31415), Peter Robinson (@drinkcoffee)
+discussions-to: https://ethereum-magicians.org/t/eip-3220-crosschain-id-specification/5446
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-10-21
+---
+
+## Simple Summary
+
+To provide better identification, description, and discovery of blockchains in cross-chain operations, this specification defines the syntax and methods for a cross-chain identification that is self-verifying within the blockchain, identifiable with forked blockchains, and embedded with essential information for the blockchain.
+
+## Abstract
+
+Crosschain operations include transfer of assets, data, message, events, and commands from one blockchain to another. In order to improve interoperability of crosschain operations, there is a need to uniquely identify source blockchain, target blockchains, and any relay chains that help facilite blockchain information transfer.  The identifier should follow a standard format so that all platforms and decentralize applications can use this identifier to represent a blockchain and also some information in the identifier to get some characteristis of the blockchain.  We name this blockchain identifier as crosschain-id and expect Ethereum community to use crosschain-id to represent any public or private blockchain.  The crosschain-id is a 32 byte hex string and with some bytes extracted from blockchain hash and some manually defined to characterize a blockchain.  We also propose a registration and lookup service to retrieve blockchain metadata from the crosschain-id.
```

Comment:
The first half of this is more appropriate for motivation section.  The abstract should be a human readable high level overview of the specification.  It shouldn't go into the "why" at all (that is for the motivation section).
```suggestion
The crosschain-id is a 32 byte hex string and with some bytes extracted from blockchain hash and some manually defined to characterize a blockchain.
We also propose a registration and lookup service to retrieve blockchain metadata from the crosschain-id.
```
You may want to go into a *little* more detail, maybe one or two sentences or one line that just shows the bit layout.